tests/resources/test_balance_entries.py

```diff
@@ -1,24 +1,18 @@
 import pytest
 
 from cuenca import BalanceEntry
-from cuenca.exc import CuencaResponseException
 
 
 @pytest.mark.vcr
 def test_balance_entry_retrieve():
     id_entry = 'TV01'
     balance_entry: BalanceEntry = BalanceEntry.retrieve(id_entry)
     assert balance_entry.id == id_entry
+    # related_transaction
```

Comment:
why is this necessary?

Reply:
Si es necesario para testear los @property de BalanceEntry